### PR TITLE
feat: ui for project proposal

### PR DESCRIPTION
**Pull request by beingnoble03**
This PR adds button for project proposal in the past projects card.

### CODE REVIEW

**Comment by beingnoble03** [comment]
@nishantwrp PTAL.

**Comment by singlamohit1** [comment]
@beingnoble03 LGTM 
@nishantwrp u can have final look and then merge

**Comment by beingnoble03** [comment]
@nishantwrp PTAL.

**Comment by nishantwrp** [comment]
Can you adjust the height of the buttons to be same?

Screenshot 2022-05-21 at 1 28 44 AM

**Comment by beingnoble03** [comment]
@nishantwrp PTAL. I fixed to height of buttons to 50px.

File: src/components/project-card.css [hunk 1]
```diff
@@ -10,4 +10,20 @@
...
+  width: 100%;
+  height: 100%;
+}
+
+.project-proposal-btn {
+  width: 100%;
+}
+
+.project-proposal-container {
+  margin-top: 10px;
```
Comment (nishantwrp) [inline src/components/project-card.css:23]: Reduce the margin to `5px`.

File: src/components/project-card.css [hunk 2]
```diff
@@ -10,4 +10,20 @@
...
+}
+
+.project-card-btn {
+  width: 100%;
+  height: 50px;
```
Comment (nishantwrp) [inline src/components/project-card.css:28]:
```suggestion
  height: 100%;
```

File: src/components/project-card.css [hunk 3]
```diff
@@ -10,4 +10,20 @@
 
 .project-card-link {
   margin: 1%;
+  display: block;
+  width: 100%;
+  height: 100%;
```
Comment (nishantwrp) [inline src/components/project-card.css:15]:
```suggestion
```

File: src/components/project-card.jsx [hunk 4]
```diff
@@ -31,11 +31,16 @@ const ProjectCard = ({ data }) => {
...
             >
-              <Button basic color="orange">
+              <Button basic color="orange" className="project-card-btn">
                 Code Submission
               </Button>
             </OutboundLink>
           </div>
+          <div className="project-proposal-container">
+            <Button color="orange" className="project-proposal-btn">
+              Project Proposal
```
Comment (nishantwrp) [inline src/components/project-card.jsx:41]: Should we change it to `Proposal` because we're not using `Project` prefix in any other links above

**Review by nishantwrp — CHANGES_REQUESTED** [review]
Thanks @beingnoble03, left a few comments. ptal!

File: src/components/project-card.jsx (hunk 4 above)
Comment (beingnoble03) [reply src/components/project-card.jsx:41]: Hmm.. Yes we can do that.

File: src/components/project-card.css (hunk 2 above)
Comment (beingnoble03) [reply src/components/project-card.css]: Done

File: src/components/project-card.css (hunk 3 above)
Comment (beingnoble03) [reply src/components/project-card.css]: done

File: src/components/project-card.css (hunk 1 above)
Comment (beingnoble03) [reply src/components/project-card.css]: Done

**Review by nishantwrp — APPROVED** [review]
lgtm, thanks @beingnoble03!

**Pull request merged**